errors/common.go

```diff
@@ -44,6 +45,14 @@ func IsInvalidFormatErr(err error) bool {
 	return IsSameError(errInvalidFormat, err)
 }
 
+func ErrUnknownModule(mod string) TMError {
```

Comment:
Need to understand TMErrors more here - seems cool, but now so many levels of separation from standard error handling - just wish we could use `pkg/errors` maybe I'm overly idealistic but it feels like the greater picture of how we pass errors around through a singular ABCI response could be improved upon (greatly?). Just brainstorming on this idea - maybe can open up a discussion on this at some point. w h a t a r e w e r e a l l y d o i n g w i t h A B C I r e s p o n s e e r r o r s.. you know